frontend/package.json

```diff
@@ -3,28 +3,23 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
-    "@fortawesome/fontawesome-svg-core": "^1.2.28",
-    "@fortawesome/free-regular-svg-icons": "^5.13.0",
-    "@fortawesome/free-solid-svg-icons": "^5.13.0",
-    "@fortawesome/react-fontawesome": "^0.1.9",
-    "@testing-library/jest-dom": "^4.2.4",
-    "@testing-library/react": "^9.5.0",
-    "@testing-library/user-event": "^7.2.1",
-    "babel-polyfill": "^6.26.0",
```

Comment:
Hmm I need to revisit this one, I thought Create React App would include the polyfills based on the `browserlist` key but we might need to import it manually maybe (https://babeljs.io/docs/en/babel-polyfill.html).

We might not need polyfills at all, I wonder what the minimum Safari version is in use in the warehouse

Reply:
Hmm so a quick look at https://caniuse.com/es6 and Safari has not needed a polyfill since v10, release in Sep 20 2016. So I think we're probably fine? If not Create React App has [their own polyfill](https://github.com/facebook/create-react-app/blob/main/packages/react-app-polyfill/README.md) we can use which doesn't tie us to Babel (I can dream that one day CRA will drop it haha)